### DISCUSSION TITLE
Path with regex not being ignored

### DISCUSSION THREAD

**Discussion by leahecole** [renovatebot/renovate#10579] [Request Help] [answered]
Hi! 

I have a repo with the following config (I know there's a warning with the bottom pytest package pattern - working on it 😄 ) . I expect it to ignore dependency updates to any constraints file under the `composer/blog` directory - for example, `composer/blog/mypost/constraints.txt` and `composer/blog/friendpost/post1/constraints.txt` should both be ignored. However, that seems to not be happening in our repo. I'm a bit confused, because I have a config in a simpler repo that is very similar and it properly ignores the constraints file. Any hints would be appreciated! 

**Problem config**
```
{
  "extends": [
    "config:base"
  ],
  "prConcurrentLimit": 0,
  "rebaseWhen": "never",
  "masterIssue": true,
  "pip_requirements": {
    "fileMatch": ["requirements-test.txt", "requirements-composer.txt", "constraints.txt", "constraints-test.txt"]
  },
  "ignorePaths" : ["composer/workflows/constraints.txt", "composer/blog/[\\S/]*constraints.txt"],
  "packageRules": [
    {
      "packagePatterns": ["pytest"],
      "separateMinorPatch": true
    },
    {
      "packagePatterns": ["pytest"],
      "updateTypes": ["patch"],
      "enabled": false
    },
    {
      "packagePatterns": ["pytest"],
      "updateTypes": ["minor", "major"]
    }
  ]
}

```

**Similar config that works fine**
```
{
  "extends": [
    "config:base",  ":preserveSemverRanges"
  ],
  "baseBranches": ["main"],
  "pip_requirements": {
    "fileMatch": ["requirements-test.txt", "samples/[\\S/]*constraints-test.txt"],
  "ignorePaths": [".pre-commit-config.yaml", "samples/[\\S/]*constraints.txt"]

  }
}
```

**Comment by rarkins** [comment, marked as answer]
`ignorePaths` does not support regex

**Reply by leahecole** [reply]
Thank you!
